modules/server/main.tf

```diff
@@ -73,242 +81,242 @@ locals {
+    INVENTORY_GCS_SUMMARY_ENABLED                       = var.inventory_gcs_summary_enabled
+    INVENTORY_EMAIL_SUMMARY_ENABLED                     = var.inventory_email_summary_enabled
+    INSTANCE_NETWORK_INTERFACE_VIOLATIONS_SHOULD_NOTIFY = var.instance_network_interface_violations_should_notify
+    IAP_VIOLATIONS_SHOULD_NOTIFY                        = var.iap_violations_should_notify
+    IAM_POLICY_VIOLATIONS_SHOULD_NOTIFY                 = var.iam_policy_violations_should_notify
+    IAM_POLICY_VIOLATIONS_SLACK_WEBHOOK                 = var.iam_policy_violations_slack_webhook
+    GROUPS_VIOLATIONS_SHOULD_NOTIFY                     = var.groups_violations_should_notify
+    FORWARDING_RULE_VIOLATIONS_SHOULD_NOTIFY            = var.forwarding_rule_violations_should_notify
+    FIREWALL_RULE_VIOLATIONS_SHOULD_NOTIFY              = var.firewall_rule_violations_should_notify
+    EXTERNAL_PROJECT_ACCESS_VIOLATIONS_SHOULD_NOTIFY    = var.external_project_access_violations_should_notify
+    ENABLED_APIS_VIOLATIONS_SHOULD_NOTIFY               = var.enabled_apis_violations_should_notify
+    CLOUDSQL_ACL_VIOLATIONS_SHOULD_NOTIFY               = var.cloudsql_acl_violations_should_notify
+    CONFIG_VALIDATOR_VIOLATIONS_SHOULD_NOTIFY           = var.config_validator_violations_should_notify
+    BUCKETS_ACL_VIOLATIONS_SHOULD_NOTIFY                = var.buckets_acl_violations_should_notify
+    BLACKLIST_VIOLATIONS_SHOULD_NOTIFY                  = var.blacklist_violations_should_notify
+    BIGQUERY_ACL_VIOLATIONS_SHOULD_NOTIFY               = var.bigquery_acl_violations_should_notify
+    AUDIT_LOGGING_VIOLATIONS_SHOULD_NOTIFY              = var.audit_logging_violations_should_notify
+    VIOLATIONS_SLACK_WEBHOOK                            = var.violations_slack_webhook
     # CSCC notifications
-    CSCC_VIOLATIONS_ENABLED = "${var.cscc_violations_enabled ? "true" : "false"}"
-    CSCC_SOURCE_ID          = "${var.cscc_source_id}"
-
+    CSCC_VIOLATIONS_ENABLED = var.cscc_violations_enabled
+    CSCC_SOURCE_ID          = var.cscc_source_id
     # Email notifications
-    EMAIL_SENDER     = "${var.forseti_email_sender}"
-    EMAIL_RECIPIENT  = "${var.forseti_email_recipient}"
-    SENDGRID_API_KEY = "${var.sendgrid_api_key}"
-
+    EMAIL_SENDER     = var.forseti_email_sender
+    EMAIL_RECIPIENT  = var.forseti_email_recipient
+    SENDGRID_API_KEY = var.sendgrid_api_key
     # Group settings
-    GROUPS_SETTINGS_MAX_CALLS                = "${var.groups_settings_max_calls}"
-    GROUPS_SETTINGS_PERIOD                   = "${var.groups_settings_period}"
-    GROUPS_SETTINGS_DISABLE_POLLING          = "${var.groups_settings_disable_polling ? "true" : "false"}"
-    GROUPS_SETTINGS_ENABLED                  = "${var.groups_settings_enabled ? "true" : "false"}"
-    GROUPS_SETTINGS_VIOLATIONS_SHOULD_NOTIFY = "${var.groups_settings_violations_should_notify ? "true" : "false"}"
+    GROUPS_SETTINGS_MAX_CALLS                = var.groups_settings_max_calls
```

Comment:
i think that all this copying of values from vars to locals is an overkill.  Maybe in the future releases we can squash all of these local variables in capital letters.